courses/templates/courses/partials/comparison_after_the_course_body.html

```diff
@@ -206,13 +206,13 @@ <h3 class="comparison__subject-heading">
                             </h3>
                         {% endif %}
 
-                        {% if course2_leo_stats %}
+                        {% if course2_leo_stats and course1.is_in_england %}
```

Comment:
Using `course1.is_in_england` instead of `course2.is_in_england`

I think this should be:
```
{% if course2_leo_stats and course2.is_in_england %}
```

Reply:
Well spotted. Corrected now.